tests/robot-tests/run_tests.py

```diff
@@ -119,6 +119,10 @@
                     dest="fail_fast",
                     action='store_true',
                     help="stop test execution on failure")
+parser.add_argument("--custom-env",
+                    dest="custom_env",
+                    default=None,
+                    help="load a custom .env file")
```

Comment:
Could add a note here that it should be placed in the robot-tests directory. Might be overkill, but does clear up any potential confusion.